### ISSUE TITLE
I want to be a polish localizer

### ISSUE THREAD

**Issue by blyedev** [sveltia/sveltia-cms#225]
Ping me when it's possible

**Comment by kyoshino** [comment]
Thanks for your interest, @blyedev! As mentioned in the [README](https://github.com/sveltia/sveltia-cms/blob/main/src/lib/locales/README.md), Sveltia CMS will soon be migrating from [svelte-i18n](https://github.com/kaisermann/svelte-i18n) to the [Fluent localization system](https://projectfluent.org/). Actually there is a Polish example on the Fluent website that shows the complexity of your language. So I think the migration is good for you. I’ll ping you as soon as l10n is ready, hopefully by the end of the year. 🙂

**Comment by kyoshino** [comment]
Sorry for keeping you waiting for so long! Sveltia CMS is now ready for localization. We’ll be adding more strings before and after the 1.0 release, but it’s a good time to start translating. Check out the [README](https://github.com/sveltia/sveltia-cms/blob/main/src/lib/locales/README.md) for instructions. Let me know if you have any questions!

**Comment by idarek** [comment]
Hi, if any help needed to this translation, I am happy to participate.

**Comment by kyoshino** [comment]
@blyedev Can you still work on this?

**Comment by blyedev** [comment]
I could but I’m a little busy at the moment, would need a few weeks to start. If Darek wants to take over the ticket I would have nothing against that.

**Comment by idarek** [comment]
I am also very busy, but if there is a place where more than one can edit strings, we probably could do this together, wherever somebody got 5 minutes of free time :)

**Comment by kyoshino** [comment]
If you prefer, I can ask Claude to prepare a draft for your review.

**Comment by idarek** [comment]
> If you prefer, I can ask Claude to prepare a draft for your review.

Can do :)

**Comment by blyedev** [comment]
Yeah to be perfectly honest I don’t expect any issues with a vibecoded translation.

**Comment by kyoshino** [comment]
It may still be difficult for even frontier models to translate short sentences. And I believe Polish in particular has a complex grammar (that’s why we chose [MF2](https://messageformat.unicode.org/) as our translation format). So, yeah, using AI is totally fine, but we’re not going to ship translations without a native review 😉

**Comment by idarek** [comment]
Ok, leave it with me :)

**Comment by idarek** [comment]
To keep updated, and avoid others duplicating (wasting time). Got all translated but need to manually review each translation, from native perspective.

**Comment by idarek** [comment]
[pl_CMS.yaml](https://github.com/user-attachments/files/30829163/pl_CMS.yaml)
[pl_UI.yaml](https://github.com/user-attachments/files/30829164/pl_UI.yaml)

Hi, Finished reviewing files. Unsure how to move with it, as I cannot upload files as described in README hence attaching here.

**Comment by kyoshino** [comment]
@idarek Do you see any error on the upload page?

**Comment by idarek** [comment]
> [@idarek](https://github.com/idarek) Do you see any error on the upload page?

<img width="791" height="172" alt="Image" src="https://github.com/user-attachments/assets/504bf471-2859-4eec-a15b-263abb0ac2b4" />
<img width="792" height="177" alt="Image" src="https://github.com/user-attachments/assets/885bcfa0-1df4-48e4-b0f1-51d2aa55d6e9" />

**Comment by kyoshino** [comment]
Hmm, looks like only I can see the upload option 😅 I _think_ you can use these pages instead:

- https://github.com/sveltia/sveltia-ui/new/main/src/lib/locales?filename=pl.yaml
- https://github.com/sveltia/sveltia-cms/new/main/src/lib/locales?filename=pl.yaml

 I’ll correct the README.

**Comment by idarek** [comment]
FYI
When click on above, I am asked to Fork repository, but than got an Error. 

<img width="1444" height="372" alt="Image" src="https://github.com/user-attachments/assets/44b308bc-ef27-49c6-ada3-127831e99980" />

I head back to both, **sveltia-ui** and **sveltia-csm** and forked them before following above links then no question about forking or error occurs.

In README it asking to fork first so there will be no issues for others :)

Added both pull requests, for UI and CMS.

**Comment by kyoshino** [comment]
Shipped Polish with [v0.182.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.182.0) 🇵🇱 

Will update the README again 😅

**Comment by blyedev** [comment]
Thanks Darek! Sorry I couldn't have been more of a help.

**Comment by kyoshino** [comment]
@idarek Hi there! I just synced the latest English strings. To save you time, I had Claude translate the new ones for you. Please review the translations and update them if necessary. [Here’s the diff](https://github.com/sveltia/sveltia-cms/commit/a0a0851d05bf79298e5ba824a14b6bad7708f871#diff-4c3049ae77cbdc2feb196141988b2d11cd93e5f4a8fbd25a79f7693f4305d11e).

Separately, Claude flagged some pluralization issues in the translations. Please [check those](https://github.com/sveltia/sveltia-cms/commit/35e6c85ba2f3fc9734b9c745da782f1389f00f1f#diff-4c3049ae77cbdc2feb196141988b2d11cd93e5f4a8fbd25a79f7693f4305d11e) as well to ensure they are accurate and contextually appropriate. If you have any questions or need further assistance, feel free to reach out. Thank you for your help!

**Comment by idarek** [comment]
> check those

All correct, can be pushed.
